target/linux/realtek/dts-5.10/rtl8393_zyxel_gs1900-48.dts

```diff
@@ -0,0 +1,315 @@
+/dts-v1/;
+
+#include "rtl839x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "zyxel,gs1900-48", "realtek,rtl838x-soc";
+	model = "Zyxel GS1900-48";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};	
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_sys: sys {
+			label = "gs1900_46:green:sys";
```

Comment:
'label' should just be "green:sys". To be forward-compatible, you can also already specify the 'color' and 'function' properties.

Reply:
Fixed.